workspace-manager-clienttests/README.md

```diff
@@ -61,6 +70,35 @@ export TEST_RUNNER_SERVER_SPECIFICATION_FILE="workspace-dev.json"
 Each service account JSON files in the resources/serviceaccounts directory of this project specifies a default file
 path for the client secret file. This default path should match where the render-config.sh script puts the secret.
 
+#### Test runner SA keys
+The test runner SA key is necessary for uploading test results to Google Buckets and BigQuery. 
+Each environment has its own test runner SA key with the right permissions for uploading test results.
+The provisioning of the test runner SA keys will be done with `Terraform` in collaboration with `devOps`.
+Additional `k8s` permission will allow test runner to manipulate a cluster for resiliency tests.
+
+Here's the vault paths of various test runner SA keys.
+
+Static environments (alpha, dev, perf, qa, staging)
```

Comment:
nit: you don't need to list the static environments twice

Reply:
Removed the duplicate reference to static env.